sdk/cosmos/azure-cosmos-spark_3-4_2-12/pom.xml

```diff
@@ -184,10 +184,5 @@
       <artifactId>jackson-module-scala_2.12</artifactId>
       <version>2.13.5</version> <!-- {x-version-update;com.fasterxml.jackson.module:jackson-module-scala_2.12;external_dependency} -->
     </dependency>
-    <dependency>
```

Comment:
Is this change necessary for Spark 3.1-3.3. as well or did we only add it to 3.4 accidentally?